### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by suniljames**
SCYTALE TEAM: IGNORE THIS

Small change for you guys to consider.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## About SPIFFE
 
-The Secure Production Identity Framework For Everyone (SPIFFE) Project defines a framework and set of
+Hello World ... The Secure Production Identity Framework For Everyone (SPIFFE) Project defines a framework and set of
```
Comment (suniljames) [inline README.md:5]: THIS IS A TERRIBLE IDEA

**Review by ajessup — CHANGES_REQUESTED** [review]
Nope.

**Comment by sejal1s** [comment]
Hi Sunil,

I am trying to open the link for Spire design document/ref architecture. But I guess link does not exist anymore. Can you please help with new path.

Path mentioned was: https://github.com/spiffe/community/blob/master/contributors/design%20docs/readme.md

Thanks.

**Comment by sejal1s** [comment]
I wish to try to setup the spire environment on Kubernetes with bare metal VMs.
Your guidance or any pointer/URL would be appreciated.

**Comment by ajessup** [comment]
Hi @sejal1s - this issue is closed and no longer monitored. For the broken link, could you file a new issue under the SPIFFE project as described [here](https://help.github.com/articles/creating-an-issue/)?

As for help on Kubernetes, we're working on better documentation for this. In the meantime, the best place to get help is in the #help channel of our Slack workspace, which is at https://spiffe.slack.com/ (you can sign up at https://slack.spiffe.io/).

**Pull request closed without merging**